project/scripts/release

```diff
@@ -325,19 +319,14 @@ else
   important git push origin --tags
 fi
```

Comment:
do we know why tags are pushed before publishing docs?

Reply:
Not sure. (eventually we can remove the tag-pushing from this script entirely anyway, since the (pushed) tag will trigger running the script in the first place)